rust/azure_iot_operations_services/src/azure_device_registry/device_discovery_gen/mod.rs

```diff
@@ -1,4 +1,4 @@
-/* Code generated by Azure.Iot.Operations.ProtocolCompiler v0.10.0.0; DO NOT EDIT. */
+/* Code generated by Azure.Iot.Operations.ProtocolCompilerLib v0.10.0.0; DO NOT EDIT. */
```

Comment:
?

Reply:
this file just hadn't gotten regenerated since the naming changed